ethsigner/signer/multiplatform/src/main/java/tech/pegasys/ethsigner/signer/multiplatform/SigningMetadataTomlConfigLoader.java

```diff
@@ -72,32 +80,84 @@
     }
   }
 
-  private Optional<FileBasedSigningMetadataFile> getMetadataInfo(final Path file) {
+  private Optional<SigningMetadataFile> getMetadataInfo(final Path file) {
+    final String filename = file.getFileName().toString();
+
     try {
       final TomlParseResult result =
           TomlConfigFileParser.loadConfigurationFromFile(file.toAbsolutePath().toString());
-      final String type = result.getTable("signing").getString("type");
+
+      final Optional<ThrowingTomlTable> signingTable =
+          getSigningTableFrom(file.getFileName().toString(), result);
+      if (signingTable.isEmpty()) {
+        return Optional.empty();
```

Comment:
nit: log error message if this happens

Reply:
The error is logged within getSigningTableFrom .... its maybe not ideal - but cleans up this code a bit - and means you get consistent error msgs for each type of metadata file.